Assets/Scripts/Bullets/BulletModel.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Bullet
+{
+    public class BulletModel
+    {
+        private float damage = 10f;
```

Comment:
try not to hard code values in model classes. instead, read these from a DefaultScriptableObject